src/serve/index.ts

```diff
@@ -236,7 +239,7 @@ export const serve = <
   TResponse extends Response = Response,
 >(
   routeFunction: RouteFunction<TInitialPayload>,
-  options?: Omit<WorkflowServeOptions<TResponse, TInitialPayload>, "useJSONContent">
+  options?: Omit<WorkflowServeOptions<TResponse, TInitialPayload>, "useJSONContent"> & ValidationOptions<TInitialPayload>
```

Comment:
we can remove ValidationOptions from here if we add them to WorkflowServeOptions